sw/device/tests/dif/dif_kmac_smoketest.c

```diff
@@ -129,5 +129,45 @@ bool test_main() {
     }
   }
 
+  // Run a SHA-3 test case with varying alignments.
+  for (size_t i = 0; i < 4; ++i) {
+    static const char *kMsg = "ABCD1234WXYZ";
+    static const size_t kMsgSize = 12;
```

Comment:
Nit: maybe use a `char[]` so that you can ARRAYSIZE it?

Reply:
Done.